DhawaAlkhreej/DhawaAlkhreej/Modules/Calculator/CalculatorVC.swift

```diff
@@ -145,3 +186,15 @@ class CalculatorVC: BaseViewController {
         }
     }
 }
+
+enum textFieldName: String {
```

Comment:
`enum` should start with uppercase like `TextFieldName` and it's better to make it private as you don't need them anywhere else

Reply:
it's not private because I am using it in the extension.